include/seqan3/alignment/configuration/align_config_output.hpp

```diff
@@ -197,7 +176,23 @@ struct output_alignment_tag : public pipeable_config_element<output_alignment_ta
  * \see seqan3::align_cfg::output_sequence1_id
  * \see seqan3::align_cfg::output_sequence2_id
  */
-inline constexpr output_alignment_tag output_alignment{};
+struct output_alignment : public pipeable_config_element<output_alignment>
+{
+    /*!\name Constructor, destructor and assignment
+     * \{
+     */
+    constexpr output_alignment() = default; //!< Defaulted.
+    constexpr output_alignment(output_alignment const &) = default; //!< Defaulted.
+    constexpr output_alignment(output_alignment &&) = default; //!< Defaulted.
+    constexpr output_alignment & operator=(output_alignment const &) = default; //!< Defaulted.
+    constexpr output_alignment & operator=(output_alignment &&) = default; //!< Defaulted.
+    ~output_alignment() = default; //!< Defaulted.
+    //!\}
+
```

Comment:
Most other config elements without a member variable to dot not have explicit ctors

```suggestion
```